jupyter-book/concluding_remarks.md

```diff
@@ -0,0 +1,261 @@
+
+The predictions may modify how the system is functions:
+  - Predicting who will benefit from an hospital stay may overcrowd some units of the hospital, and thus change the positive impact of hospitals on inpatients
+
+### Choice of the output/the labeled dataset
+
+- What we chose to predict is a very loaded choice
+- Interesting labels are often hard to get, focusing on the "easy" ways
+  of accumulating labels comes with biases
+- Our target may be a proxy of the quantity of interest
+
+
+### Biases in the data
+
+All data come with biases.
+
+- The data may not reflect the ground truth
+    - Disease monitoring is function of testing policy
+    - It may change with time, it may be uneven across the population (eg higher quality data for rich people)
+- The state of affaires may not be the desired one
+    - For equal qualifications and responsibilities, women are typically payed less than men. A learner will pick this up and amplify inequalities
+
+### Prediction models versus causal models
+
+Machine learner models are not driven by causal mechanisms.
+
+- For example people that go to the hospital die more than people who do not:
+    - Naive data analysis might conclude that hospital are bad for health
+    - The fallacy is that we are comparing different populations: people
+      who go to the hospital typically have a worse baseline health than
+      people who do not.
+- Another example: having a heart pressure greater than a threshold
+  may trigger specific care which is good. A learner will pick up
+  above-threshold heart pressure as predictor of a health improvement
+- In pure predictive settings, these informations are beneficial for the
+  predictions. However:
+ - they should not be trusted when designing interventions
+ - predictive models built on such non-causal information may be brittle
+   to changes of operational procedures
+ - interpretation is subject to caution
+
+
+### Societal impact
+
+These challenges with biases in the data, feedback loops of the
+predictions, can be very important, because **prediction models may affect
+people's lives**.
+
+Today, AI systems can be used to allocate loans, screen job applicants,
+prioritise medical treatement, help law enforcement or court decisions.
+
+If you know scikit-learn, [fairlearn](https://fairlearn.org) is a simple
+resource to understand some problems caused by a too naive application of
+machine learning methods.
+
+ML or AI can shift decision logic, power structures, operational costs
+  - As all technology, it induces changes in our society. Let us think
+    about how to make it better, even though this is a difficult question
+  - Responsible use of machine learning involves challenges at the
+    intersection of technology and society. No solution will be
+    purely technical
+
+----
+
+**Your move: choose what you will do with machine learning**
+
+
+- Machine learning drives one of the most important technological revolution of our time.
```

Comment:
```suggestion
- Machine learning drives one of the most important technological revolutions of our time.
```

Reply:
I think "revolutions" here should be in plural as it is meant to be _one out of several_ important technological revolutions